.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval"
+    ],
+    "dependencyDashboard": true,
+    "dependencyDashboardLabels": ["dependencies", "repo maintenance"],
+    "enabledManagers": ["github-actions", "npm"],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait a few days for any new package, as a precaution against malicious publishes
+    "minimumReleaseAge": "3 days",
```

Comment:
@HonkingGoose curious your thoughts on this. It's an opinion I've been spouting online without explicit validation from anyone in Dependabot/Renovate/etc. 😄

Reply:
In the Renovate org itself we use 7 days as default: https://github.com/renovatebot/.github/blob/a9b025a4302b34da7b219887630cb3bf50393ab8/default.json#L89-L95

Combined with dependency dashboard - which lets you push forward any update on demand - I think it makes sense.

Reply:
When you wait a few days, the community may discover that the package is bad. By the time you would get the Renovate PR - when `minimumReleaseAge` is reached - the bad package _may_ already have been pulled by the registry maintainers. In that perfect scenario you wouldn't even get the PR for the bad update, and instead stay on the known-good version, or get a non-malicious update after the real maintainers get back control. I don't recommend relying on this (alone) to keep you safe.

The Renovate docs mention another use for setting the `minimumReleaseAge`. [^1]

> There are a couple of uses for `minimumReleaseAge`:
>
> ...snip...
>
> #### Prevent holding broken npm packages
> 
> npm packages less than 72 hours (3 days) old can be unpublished, which could result in a service impact if you have already updated to it. Set `minimumReleaseAge` to `3 days` for npm packages to prevent relying on a package that can be removed from the registry:
> 
> ```json
> {
>   "packageRules": [
>     {
>       "matchDatasources": ["npm"],
>       "minimumReleaseAge": "3 days"
>     }
>   ]
> }
> ```

So you're right that setting `minimumReleaseAge` to `3` is "kinda-sorta" recommended by Renovate. 😄 

[^1]: [Renovate docs for `minimumReleaseAge`](https://docs.renovatebot.com/configuration-options/#minimumreleaseage)

Reply:
I'd also prefer 7 days. 3 days is tight if something is released on a Friday. :)